extensions/users-permissions/documentation/1.0.0/users-permissions-User.json

```diff
@@ -1546,6 +1546,12 @@
                     "type": "string"
                   }
                 },
+                "categories": {
```

Comment:
No, on line 1523, it shows that it's showing us the interests fields since all the fields after it are nested.  That shows that category has a relationship with interest.